projects/packages/waf/src/class-waf-standalone-bootstrap.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Handles generation and deletion of the bootstrap for the standalone WAF mode.
+ *
+ * @package automattic/jetpack-waf
+ */
+
+namespace Automattic\Jetpack\Waf;
+
+use Composer\InstalledVersions;
+use Exception;
+
+/**
+ * Handles the bootstrap.
+ */
+class WafStandaloneBootstrap {
+
+	/**
+	 * Ensures that constants are initialized if this class is used.
+	 */
+	public function __construct() {
+		$this->guard_against_missing_abspath();
```

Comment:
True, its more a question of personal preference, I like to have code that is immediately speaking to me instead of having to read a few lines to understand what is going on. One could also argue about SRP but the truth is the former is more of my reason :)